### ISSUE TITLE
"'Default' tag requires a nonempty 'Extension' attribute" on Win10

### ISSUE THREAD

**Issue by fcurella** [electronjs/windows-installer#12]
I'm trying to build an installer, but when I try to run it the installation fails with `'Default' tag requires a nonempty 'Extension' attribute`:

```
2016-02-25 08:03:31> IEnableLogger: Failed to create uninstaller registry entry: System.Xml.XmlException: 'Default' tag requires a nonempty 'Extension' attribute. Line 10, position 4.
```

I've done some googling around and it appears to be an issue with files without extensions.

The `template.nuspec` tries to create a `LICENSE` file. Removing the relevant line from `<files>` seems to resolve the problem.

**Comment by fcurella** [comment]
ref https://github.com/Squirrel/Squirrel.Windows/issues/580

**Comment by paulcbetts** [comment]
le sigh, fucking Microsoft. In the meantime, building installers on Windows (use AppVeyor if you don't have access to it) will solve this

**Comment by fcurella** [comment]
@paulcbetts to clarify, your plan is to wait until this is fixed in `Squirrel.Windows`? (not complaining, just need to know so I can plan around :) )

**Comment by paulcbetts** [comment]
@fcurella So, this bug only happens when you build a Windows installer with OS X or Linux because of a bug in NuGet (the C# version of RubyGems / npm / etc). If you build an installer with Windows as the host OS, it'll Just Work

**Comment by fcurella** [comment]
@paulcbetts that means you'd rather wait for the bug to be properly fixed in NuGet, correct?

**Comment by paulcbetts** [comment]
@fcurella No, you just need to use Windows to build your installer. We're probably going to work around this in Squirrel

**Comment by fcurella** [comment]
Thanks for clariying @paulcbetts :+1: 

FWIW for now I got it working just by editing my local copy of `template.nuspec`. I'm not familiar with developing on Windows, so that was much quicker for me.

**Comment by jprichardson** [comment]
> FWIW for now I got it working just by editing my local copy of template.nuspec. I'm not familiar with developing on Windows, so that was much quicker for me.

@fcurella can you elaborate on what you changed in `template.nuspec`? Thanks.

**Comment by fcurella** [comment]
@jprichardson I've simply removed the line mentioning `LICENSE`: https://github.com/electron/windows-installer/blob/master/template.nuspec#L22

**Comment by Mercieral** [comment]
Is there any update on if someone is working on this issue or if it will be fixed any time soon?

**Comment by develar** [comment]
@Mercieral Try electron-builder ;)
